lib/MusicBrainz/Server/Data/Statistics.pm

```diff
@@ -195,7 +195,7 @@ my %stats = (
         SQL => "SELECT " .
             join(' + ',
                  (map { "(SELECT COUNT(gid) FROM $_)" } entities_with('mbid', take => sub { my $type = shift; return shift->{table} // $type })),
-                 (map { "(SELECT COUNT(gid) FROM ${_}_gid_redirect)" } entities_with(['mbid', 'multiple'])))
+                 (map { "(SELECT COUNT(gid) FROM ${_}_gid_redirect)" } entities_with(['mbid', 'multiple'], take => sub { my $type = shift; return shift->{table} // $type })))
```

Comment:
For some reason this just assumed it should use the name of the entity, not the table like the above. I assume this exception was added specifically for collection, which didn't use to have multiple.